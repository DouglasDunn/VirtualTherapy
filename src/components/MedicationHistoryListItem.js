import React from 'react';
import { Link } from 'react-router-dom';

const MedicationHistoryListItem = ({ id, medication, dose, purpose, sideEffects, current }) => (
  <Link className="list-item" to={`/edit/${id}`}>
    <div>
      <h3 className="list-item__title">{medication}</h3>
      <h3 className="list-item__title">{dose}</h3>
      <h3 className="list-item__title">{purpose}</h3>
      <h3 className="list-item__title">{sideEffects}</h3>
      <h3 className="list-item__title">{current}</h3>
    </div>
  </Link>
);

export default MedicationHistoryListItem;
