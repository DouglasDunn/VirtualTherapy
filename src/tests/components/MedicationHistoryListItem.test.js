import React from 'react';
import { shallow } from 'enzyme';
import medicationHistories from '../fixtures/medicationHistories';
import MedicationHistoryListItem from '../../components/MedicationHistoryListItem';

test('should render MedicationHistoryListItem correctly', () => {
  const wrapper = shallow(<MedicationHistoryListItem {...medicationHistories[0]} />);
  expect(wrapper).toMatchSnapshot();
});
