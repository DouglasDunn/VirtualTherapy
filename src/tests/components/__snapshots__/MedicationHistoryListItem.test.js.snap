// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MedicationHistoryListItem correctly 1`] = `
<Link
  className="list-item"
  to="/edit/1"
>
  <div>
    <h3
      className="list-item__title"
    >
      zoloft
    </h3>
    <h3
      className="list-item__title"
    >
      10mg
    </h3>
    <h3
      className="list-item__title"
    >
      depression
    </h3>
    <h3
      className="list-item__title"
    >
      drowsiness, agitation, tiredness, loss of appetite
    </h3>
    <h3
      className="list-item__title"
    >
      yes
    </h3>
  </div>
</Link>
`;
